tests/tpchbench/src/main/scala/sparkcyclone/tpch/TPCHBenchmark.scala

```diff
@@ -270,8 +270,12 @@ object TPCHBenchmark extends SparkSessionWrapper {
     }
 
     queries.foreach { case (query, i) =>
-      if (!toSkip.contains(i)) {
```

Comment:
I guess we still want to allow skipping, if smth fails in q22 in dbgen20 , we want to run just this query instead of waiting.